keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -481,9 +486,9 @@ Describe the metrics themselves and the reasons why they weren't added (e.g., co
 implementation difficulties, etc.).
 -->
 
-Yes. It would be useful if we could see which filter plugin affected Pod's scheduling results in metrics.
-
-Issue: https://github.com/kubernetes/kubernetes/issues/110643
+It was lacking the way to see which filter plugin affected Pod's scheduling results in metrics
+(https://github.com/kubernetes/kubernetes/issues/110643),
+and we've got `plugin_evaluation_total`.
```

Comment:
It's not a lacking metric anymore. Let' move it to "How can someone using this feature know that it is working for their instance?" question - as it actually helps with that.
Please also better describe this metric.

Reply:
Done.